teradatalabs/mapr52-hive/files/setup.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+mysql_install_db
+/usr/bin/mysqld_safe &
+sleep 20
```

Comment:
@petroav sleeps are there in other setup code , i just copied from there to avoid any discrepancies